packages/react/src/search-box/SearchBoxInput.tsx

```diff
@@ -48,7 +48,7 @@ export const SearchBoxInput = forwardRef<HTMLInputElement, SearchBoxInputProps>(
 
 const useInputId = (idProp?: string) => {
 	const autoId = useId();
-	return idProp || `search-${autoId}`;
+	return idProp ?? `search-${autoId}`;
```

Comment:
[And same here with nullish coalescing](https://github.com/agriculturegovau/agds-next/pull/1685/files#r1619805454)